index.js

```diff
@@ -176,12 +179,46 @@ export default class ClearvisioAppointmentBooker {
     );
   }
 
-  createElementAndRender({parentElement, colors}) {
-    var element = document.createElement('div');
-    (parentElement || document.body).appendChild(element);
-    render(html`<${BookerComponent} store=${this.store} colors=${colors}/>`, element);
-
-    this.store.on('close', () => element.remove());
+  createElementAndRender({parentElement, colors, cssUrls}) {
+    if (this.store.get().style == 'embedded-safe') {
+      const shadowRoot = parentElement.attachShadow({ mode: 'open' });
+
+      async function loadCSSFiles(cssFiles) {
+        for(let x = 0; x < cssFiles.length; x++) {
```

Comment:
```suggestion
        for (let x = 0; x < cssFiles.length; x++) {
```

Illetve hívjuk ha lehet változót x helyett i-nek vagy index-nek ha már sima for-olunk.